### PR TITLE
Script to convert all HTML Files to Markdown Format #202

### PR DESCRIPTION
**Pull request by tejan-singh**
# Checklist:

- [X] My code follows the style guidelines(Clean Code) of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to undertand README
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests/screenshots(If any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: html-to-markdown/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### This file uses module html2markdown
```
Comment (HarshCasper) [inline html-to-markdown/README.md:1]: Create a proper Documentation Format

File: html-to-markdown/html_to_markdown.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import html2markdown
+html_text = input("Paste HTML text here (eg. <p>this is stuff <strong>stuff</strong></p>): ")
```
Comment (HarshCasper) [inline html-to-markdown/html_to_markdown.py:2]: You have to take in a HTML File and convert it to a Markdown File.

File: html-to-markdown/html_to_markdown.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+print (html2markdown.convert(html_text))
```
Comment (HarshCasper) [inline html-to-markdown/html_to_markdown.py:3]: You need to save the content in a File. Not to display it on Console

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please make the apt changes

**Comment by HarshCasper** [comment]
Project Codebase has been optimized and modified right now. Please pull the new changes to your Fork and make a fresh PR.

**Comment by tejan-singh** [comment]
I have been working on this. At least review this request and let me finish the task.

**Comment by eternal-dreamer** [comment]
Please review my pull requests . Changes have been updated.

**Pull request closed without merging**